test/libsolidity/semanticTests/array/pop/parenthesized.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+  int[] data;
+  function f() public returns (uint) {
+    data.push(1);
+    (data.pop)();
```

Comment:
maybe adding `(data.push)()` with and without parameters as well? In combination with some assertions maybe

Reply:
It does not work for `.push` because it is overloaded